cvat/apps/dataset_manager/bindings.py

```diff
@@ -535,7 +535,12 @@ def _import_shape(self, shape, parent_label_id=None):
                 self.soft_attribute_import and attrib.name not in CVAT_INTERNAL_ATTRIBUTES
             )
         ]
-        _shape['points'] = list(map(float, _shape['points']))
+
+        # FIXME: should be done in the importers, here's as a safeguard (see #1893, #1898)
```

Comment:
What is `#1893`, `#1898`? What should be fixed?

Reply:
These are numbers of the relevant issue and PR: #1893 #1898.

A fix would be complete removal of this line in favour of making sure importers are returning correct types (see changes in CVAT importer).

Reply:
For references, it's better to leave full URLs.

I'd consider this just a validation of the input data, it's not something to be fixed. It can be removed once there are type annotations, declaring the expected inputs.

Reply:
> it's not something to be fixed

Hmm, maybe TODO will suit better here. WDYT?

```suggestion
        # TODO: remove once importers are guaranteed to return correct type 
        # (see https://github.com/cvat-ai/cvat/pull/8226/files#r1695445137)
```